test/helpers/user_blocks_helper_test.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require "test_helper"
+include ApplicationHelper
```

Comment:
Well the helper itself has been excluded in `.rubocop_todo.yml` but the html_safe makes no difference to the string value and can probably be dropped in the test.